sdk/tests/conformance2/rendering/framebuffer-texture-clear.html

```diff
@@ -0,0 +1,190 @@
+  gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA8, width, height, 0, gl.RGBA, gl.UNSIGNED_BYTE, null);
+  gl.bindTexture(gl.TEXTURE_2D, null);
+  var fbo0 = gl.createFramebuffer();
+  gl.bindFramebuffer(gl.FRAMEBUFFER, fbo0);
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture0, 0);
+  gl.bindFramebuffer(gl.FRAMEBUFFER, fbo0);
+  gl.viewport(0, 0, width, height);
+  gl.disable(gl.SCISSOR_TEST);
+  gl.colorMask(true, true, true, true);
+  gl.clearColor(clearColor[0], clearColor[1], clearColor[2], clearColor[3]);
+  gl.clear(gl.COLOR_BUFFER_BIT);
+
+  var texture1 = gl.createTexture();
+  gl.bindTexture(gl.TEXTURE_2D, texture1);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.NEAREST);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.NEAREST);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
+  gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA8, width, height, 0, gl.RGBA, gl.UNSIGNED_BYTE, null);
+  gl.bindTexture(gl.TEXTURE_2D, null);
+  var fbo1 = gl.createFramebuffer();
+  gl.bindFramebuffer(gl.FRAMEBUFFER, fbo1);
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture1, 0);
+
+  var vSrc =
+        "#version 300 es\n"
+      + "precision highp float;\n"
+      + "in vec2 inPosition;\n"
+      + "in vec2 inLocalCoord;\n"
+      + "out vec2 texCoord;\n"
+      + "void main() {\n"
+      + "    texCoord = inLocalCoord;\n"
+      + "    gl_Position = vec4(inPosition, 0.0, 1.0);\n"
+      + "}\n";
+
+  var fSrc =
+        "#version 300 es\n"
+      + "precision highp float;\n"
+      + "out vec4 fragColor;\n"
+      + "uniform sampler2D uTextureSampler;\n"
+      + "in vec2 texCoord;\n"
+      + "void main() {\n"
+      + "    fragColor = texture(uTextureSampler, texCoord);\n"
+      + "}\n";
+
+  var program = wtu.loadProgram(gl, vSrc, fSrc);
+  if (!program) {
+    testFailed("Failed to create the program");
+    return;
+  }
+
+  gl.bindAttribLocation(program, 0, "inPosition");
+  gl.bindAttribLocation(program, 1, "inLocalCoord");
+
+  gl.useProgram(program);
+
+  var uTextureSamplerLoc = gl.getUniformLocation(program, "uTextureSampler");
+  gl.uniform1i(uTextureSamplerLoc, 0);
+
+  var vertexObject = gl.createBuffer();
+  gl.bindBuffer(gl.ARRAY_BUFFER, vertexObject);
+  gl.bufferData(gl.ARRAY_BUFFER, new Float32Array([
+      1.0,  1.0,
+      -1.0,  1.0,
+      -1.0, -1.0,
+      1.0,  1.0,
+      -1.0, -1.0,
+      1.0, -1.0]), gl.STATIC_DRAW);
+  gl.enableVertexAttribArray(0);
+  gl.vertexAttribPointer(0, 2, gl.FLOAT, false, 0, 0);
+
+  vertexObject = gl.createBuffer();
+  gl.bindBuffer(gl.ARRAY_BUFFER, vertexObject);
+  gl.bufferData(gl.ARRAY_BUFFER, new Float32Array([
+      1, 1,
+      0, 1,
+      0, 0,
+      1, 1,
+      0, 0,
+      1, 0]), gl.STATIC_DRAW);
+  gl.enableVertexAttribArray(1);
+  gl.vertexAttribPointer(1, 2, gl.FLOAT, false, 0, 0);
```

Comment:
I think this test should really be a WebGL 1 test if you just switch internal format from RGBA8 to RGBA, and then you can use wtu.setupTexturedQuad instead of tiu.setupTexturedQuad.  This will simplify the test code a lot (and remember to move the test from conformance2/ to conformance/)